src/conn/conn_tiered.c

```diff
@@ -542,7 +565,7 @@ __tiered_mgr_server(void *arg)
         WT_WITH_SCHEMA_LOCK(session, ret = __flush_tier_once(session, 0));
         WT_ERR(ret);
         if (ret == 0)
-            __flush_tier_wait(session);
+            WT_ERR(__flush_tier_wait(session, cfg));
```

Comment:
The `err:` exit here calls `wt_panic`.  That doesn't seem right.  We might be delayed for completely ordinary reasons, such as slow data transfer to S3.

Reply:
RIght above this the `tiered_mgr_server` always sets `timeout=0` so it should never time out. So no error is ever expected here.

Reply:
Yes, of course. My mistake.  Obviously I'm still not entirely accustomed to the way we package parameters in strings all over the place.